internal/gatewayapi/runner/runner.go

```diff
@@ -61,15 +61,24 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 		gatewayClasses := r.ProviderResources.GetGatewayClasses()
 		// Fetch the first gateway class since there should be only 1
 		// gateway class linked to this controller
-		t := &gatewayapi.Translator{
-			GatewayClassName: v1beta1.ObjectName(gatewayClasses[0].GetName()),
+		switch {
```

Comment:
nit: this is functionally correct, but I still prefer the if ... else if.... else style here, did`golangci-lint` suggest this style :) ?

Reply:
I find `switch` statements easier to read.

Reply:
I missed this before, but `switch` statements are generally more idiomatic in Go for more than one `if` statement.